app/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
+    id("androidx.navigation.safeargs")
```

Comment:
this is in kotlin, switch to groovy

Reply:
https://developer.android.com/guide/navigation/navigation-getting-started#groovy